packages/block-editor/src/components/image-editor/use-save-image.js

```diff
@@ -86,16 +83,12 @@ export default function useSaveImage( {
 				onFinishEditing();
 			} );
 	}, [
-		setIsInProgress,
 		crop,
 		rotation,
-		height,
-		width,
-		aspect,
```

Comment:
These values haven't been used since https://github.com/WordPress/gutenberg/commit/1c3c161382e7bb157778f0d50d885324953ec076#diff-f50eafec908c06fb4fc674bab8eecff3f1e2c82dbf02405564df23026c587a8bL69.

The `id` was missing dependency, and `setIsInProgress` can be omitted.